src/quacc/recipes/espresso/core.py

```diff
@@ -148,6 +152,91 @@ def relax_job(
     )
 
 
+@job
+def external_relax_job(
+    atoms: Atoms | UnitCellFilter,
```

Comment:
No need to have this. The `run_opt()` method takes an optional parameter `relax_cell`, which automatically applies the `FrechetCellFilter` to enable volume relaxations (the `FrechetCellFilter` is the improved `ExpCellFilter` which itself was the improved `UnitCellFilter`).

What this means is you can mimic what was done in the [EMT relax_job](https://github.com/Quantum-Accelerators/quacc/blob/85bb036066ff217e013ae04f9948c0c3d37b3e4c/src/quacc/recipes/emt/core.py#L57) by adding a `relax_cell: bool = False` keyword argument to the job here. Then pass that `relax_cell` value to the `base_opt_fn()`.